core/drivers/stm32_rng.c

```diff
@@ -41,6 +41,7 @@ struct stm32_rng_instance {
 	struct clk *clock;
 	unsigned int lock;
 	unsigned int refcount;
+	bool runtime;
```

Comment:
I guess the only purpose of this is to tell `stm32_rng_release()` below if this should be released or not.
A more informative name could be `keep_post_boot` or with flipped logic `release_post_boot` or something along that.